ui/pages/permissions-connect/connect-page/connect-page.tsx

```diff
@@ -159,7 +172,7 @@ export const ConnectPage: React.FC<ConnectPageProps> = ({
     [nonTestNetworkConfigurations, testNetworkConfigurations],
   );
 
-  const supportedRequestedCaipChainIds = requestedCaipChainIds.filter(
+  const _supportedRequestedCaipChainIds = requestedCaipChainIds.filter(
```

Comment:
is there something else we can name this rather than prefixing with `_` which is generally meant to denote a variable that is unused? I am guilty of doing this as well. naming is hard :_)

Reply:
Updated this to use the `supportedRequestedCaipChainIds` variable directly, lmk what u think